src/assets/scripts/client/info/AirportInfoController.js

```diff
@@ -0,0 +1,287 @@
+        this.wind = '';
+
+        /**
+         * @for AirportInfoController
+         * @property altimeter
+         * @type {Number}
+         */
+        this.altimeter = INVALID_NUMBER;
+
+        /**
+         * @for AirportInfoController
+         * @property elevation
+         * @type {String}
+         */
+        this.elevation = '';
+
+        /**
+         * @for AirportInfoController
+         * @property icao
+         * @type {String}
+         */
+        this.icao = '';
+
+        return this.init()
+                ._createChildren()
+                ._setupHandlers()
+                ._enable();
+    }
+
+    /**
+     * @for AirportInfoController
+     * @method init
+     * @chainable
+     */
+    init() {
+        this.simClockController = new SimClockController();
+        this.$template = $(AIRPORT_INFO_TEMPLATE);
+        this.$clockView = this.$template.find(INFO_VIEW_SELECTORS.CLOCK_VALUE);
+        this.$windView = this.$template.find(INFO_VIEW_SELECTORS.WIND_VALUE);
+        this.$altimeterView = this.$template.find(INFO_VIEW_SELECTORS.ALTIMETER_VALUE);
+        this.$elevationView = this.$template.find(INFO_VIEW_SELECTORS.ELEVATION_VALUE);
+
+        return this;
+    }
+
+    /**
+     * @for AirportInfoController
+     * @method _setupHandlers
+     * @chainable
+     * @private
+     */
+    _setupHandlers() {
+        this.onAirportChangeHandler = this.onAirportChange.bind(this);
+
+        return this;
+    }
+
+    /**
+     * Set initial element references
+     *
+     * Should be run once only on instantiation
+     *
+     * @for StripViewModel
+     * @method _createChildren
+     * @chainable
+     * @private
+     */
+    _createChildren() {
+        this.$element.append(this.$template);
+
+        return this;
+    }
+
+    /**
+     * Enable all event handlers
+     *
+     * @for AirportInfoController
+     * @method _enable
+     * @chainable
+     * @private
+     */
+    _enable() {
+        this._eventBus.on(EVENT.AIRPORT_CHANGE, this.onAirportChangeHandler);
+        this.onAirportChange();
+
+        return this;
+    }
+
+    /**
+     * Disable all event handlers and destroy the instance
+     *
+     * @for AirportInfoController
+     * @method _disable
+     * @chainable
+     * @private
+     */
+    _disable() {
+        this.$element = null;
+        this.$template = null;
+        this.simClockController = null;
+        this.airportInfoController = null;
+
+        this._eventBus.off(EVENT.AIRPORT_CHANGE, this._onAirportChangeHandler);
+
+        return this;
+    }
+
+    /**
+     * @for AirportInfoController
+     * @method reset
+     * @chainable
+     */
+    reset() {
+        this.wind = null;
+        this.altimeter = null;
+        this.elevation = null;
+        this.icao = null;
+
+        return this;
+    }
+
+    /**
+     * Updates the information taken from the AirportModel: the wind, the altimeter,
+     * and the elevation. Triggered on airport change.
```

Comment:
another spot for a little more documentation.  this says:
>Triggerd on airport change

which, based on what I called out above, leads to the question: _'then why is this method called in `._enable()`'_?

Reply:
no longer called from `enable`